CHANGELOG.md

```
@@ -1,45 +1,29 @@
 # [0.7.0](https://github.com/VilnaCRM-Org/core-service/compare/v0.6.0...v0.7.0) (2025-12-11)
 
-
 ### Features
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)